GenericCamera/v3/_init.yaml

```diff
@@ -1,10 +1,11 @@
 s3instance: null
+image_service_url: "http://localhost/imageService"
 instances:
   - sal_index: 1
     ip: "127.0.0.1"
     port: 5013
-    file_name_format: "{timestamp}-{index}-{total}"
     camera: "Simulator"
+    require_image_service: false
```

Comment:
Uhm.. I didn't realized this was in the "instances" level. If you do it one level up you  can get rid of duplicating this configuration everywhere else.. 

I'll leave that to your discretion..